sites/svelte.dev/content/blog/2023-05-01-whats-new-in-svelte-may-2023.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "What's new in Svelte: May 2023"
+description: "TBD"
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+TBD... Svelte Hackathon winner goes here too.
+
+## What's new in Svelte
+- `style` blocks now support CSS `@container` queries (**3.58.0**)
+- `bind:innerText` is now available for `contenteditable` elements (**3.58.0**)
+- A new accessability warning, `a11y-interactive-supports-focus` will now warn when interactive elements are not focusable (**3.58.0**)
+
+For all the changes to the Svelte compiler, including unreleased changes, check out the [CHANGELOG](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md).
+
+## What's new in SvelteKit
+- When hot module reloading (HMR) is enabled, the page will reload after an error is fixed (**1.14.0**, [#9497](https://github.com/sveltejs/kit/pull/9497))
+- Two pages generated into the same page can now be loaded simultaneously in "embedded" mode (**1.15.7**, [#9610](https://github.com/sveltejs/kit/pull/9610))
+
+
```

Comment:
there's a new release of `vite-plugin-svelte` this month: https://github.com/sveltejs/vite-plugin-svelte/blob/main/packages/vite-plugin-svelte/CHANGELOG.md#210

it's got a new feature that logs warnings for packages that aren't setup correctly. might be worth mentioning it so that people are aware that will be happening